cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -96,12 +106,32 @@ func startMizarPodController(ctx *ControllerContext, grpcHost string, grpcAdapto
 	controllerName := "mizar-pod-controller"
 	klog.V(2).Infof("Starting %v", controllerName)
 
-	go controllers.NewMizarPodController(
-		ctx.InformerFactory.Core().V1().Pods(),
-		ctx.ClientBuilder.ClientOrDie(controllerName),
-		grpcHost,
-		grpcAdaptor,
-	).Run(mizarPodControllerWorkerCount, ctx.Stop)
+	podKubeconfigs := ctx.ClientBuilder.ConfigOrDie(controllerName)
+
+	crConfigs := *podKubeconfigs
+	for _, cfg := range crConfigs.GetAllConfigs() {
+		cfg.QPS *= 5
+		cfg.Burst *= 10
+		cfg.ContentType = "application/json"
+		cfg.AcceptContentTypes = "application/json"
+	}
+	networkClient := arktos.NewForConfigOrDie(&crConfigs)
+	//TO DO: all mizar controllers here should share one informerFactory
+	//       other than every mizar controller starts one informerFactory
+	informerFactory := externalversions.NewSharedInformerFactory(networkClient, 0*time.Minute)
```

Comment:
0 should be fine. No need to add *time.Minute

Reply:
yes, easy change, will do.

Reply:
done.